### DISCUSSION TITLE
Using Quarto for my website - loving it!

### DISCUSSION THREAD

**Discussion by beyond2060** [quarto-dev/quarto-cli#5594] [Show and Tell]
I have been using Quarto for my personal website, and I'm loving it. I have modelled my pages on the Tufte layout, and I've had several people comment that they really like the sidenotes. I've been creating mildly amusing illustrated essays about AI:

https://www.beyond2060.com/ai-ethics/ 
https://www.beyond2060.com/ai-work-zombie-apocalypse/ 

I think the site would be better if I use the blog functionality - I'll fix that at some point. 

I'm also using Quarto with VS Code as my PIM. I've tried many PIMs over the years (most recently Notion and Craft) and I think Quarto is the best option so far. I will write a blog post about it when I get a moment.

**Comment by pglpm** [comment]
It looks really good! Thank you for sharing about your experience. I indeed wanted to ask: Quarto can also be used on a personally hosted website, not just GitHub etc?

**Comment by beyond2060** [comment]
Thanks for your comments. 
Yes. I edit the website using the VS Code Quarto extension, which is very good. I then copy the code to my host. You can use anything to host. I use https://tiiny.host, which means I can just zip the site up and drag/drop it to update my site. James

**Reply by pglpm** [reply]
Thank you for the explanation, James!
